### PR TITLE
Add Store options in store creation payload

### PR DESCRIPTION
**Pull request by pedromcp90**
Now we send the options in the Store instead of sending them in the DataSource.

Notice: 
To be merged after https://github.com/doofinder/doomanager/pull/2832

### CODE REVIEW

**Review by sofia-doofinder — APPROVED** [review]

**Comment by sofia-doofinder** [comment]
good job! :)

File: Controller/Adminhtml/Integration/CreateStore.php [hunk 1]
```diff
@@ -146,9 +150,7 @@ public function generateSearchEngineData($websiteID)
                             [
                                 'type' => 'magento2',
                                 'options' => [
-                                    'url' => $this->urlInterface->getBaseUrl() . 'rest/' . $store->getCode() . '/V1/',
-                                    'token' => $integrationToken,
-                                    'website_id' => $store->getWebsiteId(),
+                                    'url' => $this->urlInterface->getBaseUrl() . 'rest/' . $store->getCode() . '/V1/'
```
Comment (Aitor-Corrales) [inline Controller/Adminhtml/Integration/CreateStore.php:153]: We need to make sure that the website_id is being obtained in doofeeds from the store in order to obtain the corresponding products from it, if we make this change.

Comment (Aitor-Corrales) [reply Controller/Adminhtml/Integration/CreateStore.php:153]: We talked over it, and this is getting handled in [this](https://github.com/doofinder/doomanager/pull/2832) PR

**Review by Aitor-Corrales — APPROVED** [review]

**Comment by pedromcp90** [comment]
Thanks @sofia-doofinder !!

**Pull request merged**